buildSrc/src/minimumRuntime/java/org/elasticsearch/gradle/info/BuildParams.java

```diff
@@ -155,6 +154,7 @@ public void reset() {
                 .filter(f -> Modifier.isStatic(f.getModifiers()))
                 .forEach(f -> {
                     try {
+                        f.setAccessible(true);
```

Comment:
can you please add a comment explaining why this is needed?

Reply:
I've added a comment to explain the need for this.